server/server_test.go

```diff
@@ -250,12 +250,12 @@ func runTestLoadData(c *C) {
 		err = os.Remove(path)
 		c.Assert(err, IsNil)
 	}()
-	_, err = fp.WriteString(`
-xxx row1_col1	- row1_col2	1abc
-xxx row2_col1	- row2_col2	
-xxxy row3_col1	- row3_col2	
-xxx row4_col1	- 		900
-xxx row5_col1	- 	row5_col3`)
+	_, err = fp.WriteString("\n" +
```

Comment:
There are trailing whitespace (tab) in the content, which might be trimmed in some editors with configurations like "trim trailing whitespace on save". To prevent such issues, I wrap them with quotes.